src/services/EcoBridge/Socket/SocketBridge.ts

```diff
@@ -468,42 +402,214 @@ export class SocketBridge extends EcoBridgeChildBase {
 
     const { toAmount, serviceTime, totalGasFeesInUsd, routeId } = bestRoute
 
+    const formattedToAmount = Number(formatUnits(toAmount, result.toAsset.decimals)).toFixed(2).toString()
```

Comment:
Every bridge formats output to 2 decimals and in some cases it will affect the final output (e.g small amounts). Also displaying output like this `0.000000000000321` can be problematic so we rounded it.
@0xVenky  Do you have any idea how to display it correctly ?

Reply:
I see the point. In the Limit Order implementation, I created two state variables -- one for the UI and one for handling the logic underneath. It may require major work, so let's consider it in another sprint.